infrastructure/modules/serverless/data.tf

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2023, Xgrid Inc, https://xgrid.co
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#        http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# tflint-ignore: terraform_unused_declarations
+data "aws_kms_alias" "check_existing_kms" {
+  name = "alias/${var.namespace}-kms-key"
+}
```

Comment:
```suggestion
}

```

Reply:
Added in 7da0f0c5d8be21ed38fb14a169ce80ea5a3c8c25